### PR TITLE
chore: rebrand to gosplitsies

### LINKED ISSUE #7: Rebrand to `gosplitsies`

### PR DESCRIPTION
**Pull request by JosephMart**
fixes #7

### CODE REVIEW

File: REUSE.toml [hunk 1]
```diff
@@ -13,5 +13,5 @@ path = [
     "ui/package-lock.json",
 ]
 precedence              = "override"
-SPDX-FileCopyrightText  = "2024 Joseph Martinsen <joseph@martinsent.com>"
+SPDX-FileCopyrightText  = "2024 Joseph Martinsen <joseph@martinsen.com>"
```
Comment (JosephMart) [inline REUSE.toml:16]: Unrelated cleanup

**Review by tristan957 — APPROVED** [review]
